p2p/protocol.go

```diff
@@ -457,15 +457,8 @@ func (c *conn) handleNewPooledTransactionHashes(ctx context.Context, version uin
 	var name string
 
 	switch version {
-	case 66, 67:
```

Comment:
fyi @minhd-vu, I had to drop support for `eth/66` and `eth/67` since *it looks like* it's no longer supported by geth.